src/runtime/object.cc

```diff
@@ -24,6 +24,7 @@
 #include <tvm/runtime/object.h>
 #include <tvm/runtime/registry.h>
 
+#include <iostream>
```

Comment:
is this change necessary?

Reply:
This is to fix another compilation error with clang:
```
stderr: tvm/tvm/src/runtime/object.cc:171:14: error: missing '#include <iostream>'; declaration of 'cerr' must be imported from module 'third_party_libgcc_std.iostream' before it is required
        std::cerr << '[' << info.index << "] " << info.name
             ^
third-party-buck/platform007/build/libgcc/include/c++/7.3.0/iostream:62:18: note: previous declaration is here
  extern ostream cerr;          /// Linked to standard error (unbuffered)
                 ^
1 error generated.
    When running <c++ preprocess_and_compile>.
    When building rule //tvm:tvm#compile-pic-object.cc.od8dbd3a8,platform007-clang.
```
Should I split it into another PR instead?